### ISSUE TITLE
set up tox

### ISSUE THREAD

**Issue by nabicht** [nabicht/SimpleTaskQueue#82]
This makes it easier to run tests in multiple virtual environments. 

When setting this up also make sure i have a virtual env for at least:

1. 3.6
2. 2.7

Dependent on #67

**Comment by aggmoulik** [comment]
You don't have any setup.py file for the tox implementation.

**Comment by nabicht** [comment]
Agreed. I don’t have a setup.py file (yet). I will. This is really dependent on that task. Issue editors to reflect dependency. Thanks!
